invenio_data/base/fixtures/accounts.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of CERN Analysis Preservation Framework.
+# Copyright (C) 2014, 2015 CERN.
+#
+# CERN Analysis Preservation Framework is free software; you can
+# redistribute it and/or modify it under the terms of the GNU General
+# Public License as published by the Free Software Foundation; either
+# version 2 of the License, or (at your option) any later version.
+#
+# CERN Analysis Preservation Framework is distributed in the hope that
+# it will be useful, but WITHOUT ANY WARRANTY; without even the
+# implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+# PURPOSE.  See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this software; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307,
+# USA.
+
+from fixture import DataSet
+
+from flask import current_app
+
+from invenio_accounts import fixtures as default
+
+from invenio_groups.models import PrivacyPolicy, SubscriptionPolicy
+
+
+class UserData(DataSet):
+
+    """User data."""
+
+    class admin:
+        id = 1
+        email = current_app.config.get('CFG_SITE_ADMIN_EMAIL')
+        password = None
+        note = '1'
+        nickname = 'admin'
+
+    class marco:
+        id = 2
+        email = 'marco@crepererum.net'
+        password = None
+        note = '1'
+        nickname = 'marco@crepererum.net'
+
+    class pamfilos:
+        id = 3
+        email = 'pamfilos.fokianos@cern.ch'
+        password = None
+        note = '1'
+        nickname = 'pamfilos.fokianos@cern.ch'
+
+    class tibor:
+        id = 4
+        email = 'tibor.simko@cern.ch'
+        password = None
+        note = '1'
+        nickname = 'tibor.simko@cern.ch'
+
+
+class GroupData(DataSet):
+
+    class support:
+        id = 1
+        name = 'analysis-preservation-support'
+        description = 'CERN Analysis Preservation Support Team'
+        is_managed = True
+        privacy_policy = PrivacyPolicy.PUBLIC
+        subscription_policy = SubscriptionPolicy.CLOSED
+
+    class development:
+        id = 2
+        name = 'DAPF-development'
+        description = 'People involved with the development of the CERN Analysis Preservation Platform'
```

Comment:
Instead of (old) `DAPF-development`, please check using (new) `analysis-preservation-development` e-group name.  

If the latter is an alias to the former, then let's try to swap them around in e-groups in order to have nice homogeneous naming `analysis-preservation-{support,development}`. CC @suenjedt